docs/Updating-gene-and-gene_alias-tables.md

```diff
@@ -1,21 +1,27 @@
+# Updating the gene names and aliases tables
 This manual is intended for users that have knowledge about the structure of the cBioPortal seed database.
 
-When loading studies into cBioPortal it is possible for warnings to occur that are caused by an outdated seed database. Gene symbols can be deprecated or be assigned to a different Entrez Gene in a new release. Also Entrez Gene IDs can be added. This markdown explains how to update the seed database, in order to use the most recent Entrez Gene IDs. 
+When loading studies into cBioPortal it is possible for warnings to occur that are caused by an outdated seed database. Gene symbols can be deprecated or be assigned to a different Entrez Gene in a new release. Also Entrez Gene IDs can be added. This markdown explains how to update the seed database, in order to use the most recent Entrez Gene IDs.
 
-The cBioPortal scripts package provides a method to update the `gene` and `gene_alias` tables. This requires the latest version of the NCBI Gene Info: [click here for human](ftp://ftp.ncbi.nih.gov/gene/DATA/GENE_INFO/Mammalia/Homo_sapiens.gene_info.gz) and [here for mouse](ftp://ftp.ncbi.nih.gov/gene/DATA/GENE_INFO/Mammalia/Mus_musculus.gene_info.gz).
+The cBioPortal scripts package provides a method to update the `gene` and `gene_alias` tables. This requires the latest version of the NCBI Gene Info.
 
-## Updating the gene names and aliases
+### Human genes
+Homo_sapien.gene_info.gz\
+ftp://ftp.ncbi.nih.gov/gene/DATA/GENE_INFO/Mammalia/Homo_sapiens.gene_info.gz
 
+### Mouse genes
+Mus_musculus.gene_info.gz\
+ftp://ftp.ncbi.nih.gov/gene/DATA/GENE_INFO/Mammalia/Mus_musculus.gene_info.gz
+
+## MySQL steps
 Execute these steps in case you want to reset your database to the most recent genes list from NCBI.
 
-1- Remove all studies from your installation. You can use the [study removal tool](Development%2C-debugging-and-maintenance-mode-using-cbioportalImporter#deleting-a-study). Also empty tables `mutation_event` and `cna_event`
+1- Remove all studies from your installation. You can starting a new MySQL database with the previous seed database or use the [study removal tool](Development%2C-debugging-and-maintenance-mode-using-cbioportalImporter#deleting-a-study). In that latter case, also empty tables `mutation_event` and `cna_event`:
```

Comment:
typo: "You can starting..." 
Also: add link to "seed database" -> is nice if this points to the seed db documentation